src/chocolatey.tests/infrastructure.app/services/TemplateServiceSpecs.cs

```diff
@@ -393,7 +394,7 @@ public void should_generate_all_files_and_directories()
             }
 
             [Fact]
-            [WindowsOnly]
+            [Platform(Exclude="Unix,Linux,MacOsX")]
```

Comment:
Should be no changes to this file once you've fixed WindowsOnly Attribute.

Reply:
Done